sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java

```diff
@@ -13,6 +12,7 @@
  * Error attachment utilities.
  */
 @SuppressWarnings("WeakerAccess")
+/* TODO (getErrorAttachment): Re-enable error attachment in javadoc when the feature becomes available. Add @ before link. */
 public final class ErrorAttachments {
```

Comment:
Javadoc tool will still produce documentation for any public class, I really don't understand the changes on this class. We should rather change visibility of this class to package private and make necessary adjustement (visibility tricks) if we use it for testing somewhere.